prac_05/hex_colours.py

```diff
@@ -1,5 +1,5 @@
 """
-CP1404/CP5632 Practical
+CP1404 Practicals
```

Comment:
This code doesn't show if the user input an invalid input other than "". Add an if statement to check if the user inputs the correct input.